src/app/beer_garden/requests.py

```diff
@@ -757,6 +757,23 @@ def handle_event(event):
             for request_event in request_map:
                 request_map[request_event].set()
 
+        # If the local garden is trying to Request an event and it fails, mark the request
+        # with an error message
+        elif (
+            event.name
+            in (
+                Events.GARDEN_UNREACHABLE.name,
+                Events.GARDEN_ERROR.name,
+                Events.GARDEN_NOT_CONFIGURED.name,
+            )
+            and event.payload.model_type == "Request"
+        ):
+            complete_request(
+                event.payload.model.id,
+                status="ERROR",
+                error_class=event.error_message,
+            )
+
```

Comment:
I really don't like doing this in the garden module. Can we just call `complete_request` directly in the error handling of the `forward` function?